app/src/main/java/com/teamagam/gimelgimel/app/map/esri/EsriGGMapView.java

```diff
@@ -0,0 +1,203 @@
+import com.teamagam.gimelgimel.app.map.viewModel.gestures.OnMapGestureListener;
+import com.teamagam.gimelgimel.domain.layers.entitiy.VectorLayerPresentation;
+import com.teamagam.gimelgimel.domain.map.entities.mapEntities.GeoEntity;
+import com.teamagam.gimelgimel.domain.notifications.entity.GeoEntityNotification;
+
+
+public class EsriGGMapView extends MapView implements GGMapView {
+
+    private static final AppLogger sLogger = AppLoggerFactory.create();
+    private static final SpatialReference WGS_84_GEO = SpatialReference.create(
+            SpatialReference.WKID_WGS84
+    );
+
+    private OnReadyListener mOnReadyListener;
+    private ArcGISTiledMapServiceLayer mBaseLayer;
+    private GraphicsLayerGGAdapter mGraphicsLayerGGAdapter;
+
+    public EsriGGMapView(Context context) {
+        super(context);
+        init();
+    }
+
+    public EsriGGMapView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        init();
+    }
+
+    @Override
+    public void setGGMapGestureListener(OnMapGestureListener onMapGestureListener) {
+
+    }
+
+    @Override
+    public void lookAt(PointGeometryApp point) {
+        Point esriPoint = transformToEsri(point);
+        centerAt(esriPoint, true);
+    }
+
+    @Override
+    public void lookAt(PointGeometryApp point, float cameraHeight) {
+        Point esriPoint = transformToEsri(point);
+        zoomToResolution(esriPoint, cameraHeight);
+    }
+
+    @Override
+    public void updateMapEntity(GeoEntityNotification geoEntityNotification) {
+        GeoEntity entity = geoEntityNotification.getGeoEntity();
+        int action = geoEntityNotification.getAction();
+        switch (action) {
+            case GeoEntityNotification.ADD:
+                mGraphicsLayerGGAdapter.draw(entity);
+                break;
+            case GeoEntityNotification.REMOVE:
+                mGraphicsLayerGGAdapter.remove(entity.getId());
+                break;
+            case GeoEntityNotification.UPDATE:
+                mGraphicsLayerGGAdapter.remove(entity.getId());
+                mGraphicsLayerGGAdapter.draw(entity);
+                break;
+            default:
+                sLogger.w("Update map entity could not be done because of unknown action code: "
+                        + action);
+        }
+    }
+
+    @Override
+    public void setOnEntityClickedListener(MapEntityClickedListener mapEntityClickedListener) {
+
+    }
+
+    @Override
+    public void showVectorLayer(VectorLayerPresentation vectorLayerPresentation) {
+
+    }
+
+    @Override
+    public void hideVectorLayer(String vectorLayerId) {
+
+    }
+
+    @Override
+    public void setOnReadyListener(OnReadyListener onReadyListener) {
+        mOnReadyListener = onReadyListener;
+    }
+
+    private void init() {
+        setBasemap();
+
+        setAllowRotationByPinch(true);
+
+        logTappedCoordinates();
+    }
+
+    private void setExtentOverIsrael() {
```

Comment:
I know it sucks, but I reordered them in a later commit.
Changing it will increase rebase conflicts